ports/glutin/window.rs

```diff
@@ -1268,12 +1269,33 @@ impl WindowMethods for Window {
                     self.event_queue.borrow_mut().push(WindowEvent::Reload);
                 }
             }
+            //On windows 8.1 when pressing ctrl + anything, ch = Some('\u{unicode_value}')
+            //so matching ch with Some('l') doesn't work here
+            //Instead we match with what's inside the key variable
+            (CMD_OR_CONTROL, _, Key::L) => {
+                if let Some(true) = PREFS.get("shell.builtin-key-shortcuts.enabled").as_boolean() {
+                    let user_input: String;
+                    //TODO: 
+                    // 1) make it work when clicking on "validate" button - Currently only works when pressing enter
```

Comment:
Maybe it has to do with the fact we're blocking the main thread.
On Mac, the buttons work, but we get the beatchball.

Maybe this code should run in a different thread.